docs/components/arm/lite6.md

```diff
@@ -102,6 +102,41 @@ See [the frame system service](/services/frame-system/) for more information on
 
 {{< readfile "/static/include/components/test-control/arm-control.md" >}}
 
+### DoCommand
+
+Execute the model-specific commands `set_speed` and `set_acceleration` that are not otherwise defined by the component API.
+
+{{< tabs >}}
```

Comment:
these tabs should go.

Reply:
Is it ok to keep them since we are adding examples for the three sdks now?

Reply:
We really should not be duplicating method docs here. Since you've already created the code samples keep them, but we probably shouldn't take the time to do them in the future

Reply:
Ok, sorry for the misunderstanding